### PR TITLE
Payment in client

### PR DESCRIPTION
**Pull request by samigeorgiev**

### CODE REVIEW

File: client/src/pages/ExecutePayment.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    const history = useHistory();
+
+    const { executePayment } = methods;
+    useEffect(() => {
+        const searchParams = new URLSearchParams(location.search);
+        const paymentId = searchParams.get('paymentId');
+        const payerId = searchParams.get('PayerID');
+        console.log(paymentId, payerId);
+        const request = new ExecutePaymentRequest();
+        request.setPaymentid(paymentId);
```
Comment (toniyanchev) [inline client/src/pages/ExecutePayment.jsx:23]: why is not camel case

File: client/src/pages/ExploreCourses.jsx [hunk 2]
```diff
@@ -51,11 +57,25 @@ const Courses = props => {
     }, [contentManagementResponse]);
 
     const { response: accountOperationsResponse } = accountOperationsState;
+    const { startPayment } = paymentMethods;
     useEffect(() => {
         if (accountOperationsResponse) {
-            setIsModalShown(false);
+            const request = new StartPaymentRequest();
+            request.setEnrollmentid(
```
Comment (toniyanchev) [inline client/src/pages/ExploreCourses.jsx:64]: here too

File: client/src/pages/ExecutePayment.jsx (hunk 1 above)
Comment (samigeorgiev) [reply client/src/pages/ExecutePayment.jsx:23]: This is auto-generated source from grpc. The problem is in the library

File: client/src/pages/ExploreCourses.jsx (hunk 2 above)
Comment (samigeorgiev) [reply client/src/pages/ExploreCourses.jsx:64]: same

File: client/src/pages/ExecutePayment.jsx (hunk 1 above)
Comment (toniyanchev) [reply client/src/pages/ExecutePayment.jsx:23]: all clear now

**Pull request merged**